tests/testthat/test-mock.r

```diff
@@ -116,9 +116,11 @@ test_that("can mock hidden functions", {
 })
 
 test_that("can mock if package is not loaded", {
-  if ("package:devtools" %in% search())
-    skip('devtools is loaded')
-  with_mock(`devtools::add_path` = identity, expect_identical(devtools::add_path, identity))
+  if ("package:boot" %in% search())
+    skip('boot is loaded')
+  if (!length(find.package("boot", quiet=TRUE)))
```

Comment:
Can you please use `requireNamespace("boot", quietly = TRUE)`?

But is it even necessary to test? I think only broken installs will not have boot. But won't you also have to add boot to suggests in order to avoid R CMD check warnings?